covenant.cabal

```diff
@@ -88,19 +91,22 @@ common bench-lang
 library
   import: lang
   exposed-modules:
+    -- FIXME: Re-export the important bits from somewhere else or expose this module (ask Koz which is better)
     Control.Monad.Action
     Control.Monad.HashCons
     Covenant.ASG
     Covenant.Constant
     Covenant.DeBruijn
     Covenant.Index
+    Covenant.Internal.Data
```

Comment:
There should probably be a `Covenant.Data` exporting a safe and sensible API.

Reply:
Yeah that is probably right, I end up having to export it for tests. Will change.